packages/0x.js/src/index.ts

```diff
@@ -32,6 +31,7 @@ export {
     ContractEventArg,
     LogWithDecodedArgs,
     Order,
+    Provider,
```

Comment:
Don't forget to add `Provider` to the public types list in `zero_ex_js_documentation.ts`